arklib/src/lib.rs

```diff
@@ -0,0 +1,76 @@
+#[macro_use] extern crate log;
+#[cfg(target_os="android")]
+#[allow(non_snake_case)]
+pub mod android {
+    extern crate jni;
+
+    use jni::JNIEnv;
+    use jni::objects::{JString, JClass};
+    use jni::sys::{jlong};
+    
+    use std::fs;
+    use std::path::Path;
+    use std::io::{BufReader, BufRead};
+    use crc32fast::Hasher;
+
+    extern crate android_logger;
+    use log::Level;
+    use android_logger::Config;
+
+    #[no_mangle]
+    pub unsafe extern fn Java_space_taran_arknavigator_mvp_model_repo_index_ResourceMeta_00024Companion_helloWorld(
```

Comment:
Can be removed

Reply:
Thank you for that comment; This test function was also deleted in the latest commit.